projects/prysm-bn-validator-api.md

```diff
@@ -0,0 +1,69 @@
+# Prsym Beacon Node - Validator API
+
+Implementing the standard Ethereum Beacon API for validator clients in the Prysm beacon node
+
+## Motivation
+
+> When Prysm first started we advocated strongly for gRPC to be used for validator <> beacon communication, but in the end the standard enshrined an OpenAPI spec known as the Beacon Node API ([here](https://ethereum.github.io/beacon-APIs/) and [here](https://github.com/ethereum/beacon-APIs/)).
+
+In production, the Prysm beacon node exposes a GRPC API for consumption by the Prysm validator.
+
+As a result of the standard Ethereum [spec](https://ethereum.github.io/beacon-APIs/#/Validator), this implementation is incompatible with other validator clients.
```

Comment:
I think this is the correct URL: https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/ValidatorRequiredApi

Reply:
although these endpoints do not exactly correspond to what we have to implement

Reply:
updated